composer.json

```diff
@@ -24,7 +25,7 @@
         "illuminate/support": "^9.48 || ^10.0",
         "illuminate/view": "^9.48 || ^10.0",
         "nikic/php-parser": "^4.13",
-        "orchestra/testbench": "^7.19 || ^8.0",
+        "orchestra/testbench-core": "^7.35 || ^8.15",
```

Comment:
>^7.35 || ^8.15"

Do these versions contain for required functionality, or are they just to enforce new versions? 

Note, this plugin built to support old versions

Reply:
Both versions contain conflict restrictions (using composer.json) to ensure Testbench Core 7 will only be used with Laravel 9 and Testbench Core 8 only used with Laravel 10.